lib/openssl/signature_algorithm/eddsa.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require "ed25519"
```

Comment:
Given we are not including this as a runtime dependency in the `gemspec` file (which I think is fine), users need to explicitly include this in their `Gemfile` for `EdDSA` to work.

We can do something a tiny bit more sophisticated to give the user a more helpful error message here if the gem not present. A good example is what `activesupport` does for the cache backends. See e.g. `RedisCacheStore` https://github.com/rails/rails/blob/fbe2433be6e052a1acac63c7faf287c52ed3c5ba/activesupport/lib/active_support/cache/redis_cache_store.rb#L3-L10.